spec/requests/static_pages_spec.rb

```diff
@@ -3,27 +3,27 @@
 RSpec.describe "StaticPages", type: :request do
   describe "#home" do
     it "should have the title Home" do
-      visit '/static_pages/home'
+      visit root_path
 
       expect(page).to have_title('Expenses | Home')
     end
 
     it "should have basic info about the app" do
-      visit '/static_pages/home'
+      visit root_path
 
-      expect(page).to have_content('Tracking expenses made easy')
+      expect(page).to have_content('Welcome to the Expenses App where tracking your expenses are made easy')
```

Comment:
Line is too long. [108/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.